packages/aws/src/clients/cloudformation.client.ts

```diff
@@ -60,18 +60,18 @@ export class CloudformationClient implements CloudformationClientInterface {
      * Gets the description and all its details from a Cloudformation stack.
      * @param stackName The stack name to get the.
      */
-    async getStackDescription(stackName: string): Promise<Stack> {
+    async getStackDescription(stackName: string): Promise<Stack | undefined> {
         this.logHandler.debug("CLOUDFORMATION CLIENT - Getting stack information", {stackName}, AwsModuleKeyname);
         const command = new DescribeStacksCommand({
             StackName: stackName,
         })
         try {
             const response: DescribeStacksCommandOutput = await this.getClient().send(command);
             if(!response.Stacks || response.Stacks.length < 1){
-                throw new Error("No stacks were returned from cloudformation");
+                return undefined;
```

Comment:
I think maybe we should log a warning here as well ?

Reply:
No because it's the only way to know if a stack already exists, it's to query it.